examples/apis/example.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name" : "Example",
+    "slug" : "example",
```

Comment:
I'm not sure... We could probably do that, but do you think it makes sense to call it ID? We could probably just have the `name` property instead, this way we make sure the name is unique and also it make things simpler for whoever is configuring the API